src/dune_rules/melange/melange_binary.ml

```diff
@@ -10,7 +10,7 @@ let where =
     let* _ = Build_system.build_file bin in
     let+ where =
       Memo.of_reproducible_fiber
-      @@ Process.run_capture_line ~display:!Clflags.display Process.Strict bin
+      @@ Process.run_capture_line ~display:Quiet Process.Strict bin
```

Comment:
Here we query the location of melange. This is internal and does not need to be reported to the user.